src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -447,4 +467,45 @@ private static function recursiveRmdir($directory)
         }
         rmdir($directory);
     }
+
+    /**
+     * Ask if user would like to remove composer.lock entry from .gitignore file.
+     *
+     * @param IOInterface $io
+     * @return bool
+     */
+    private static function requestExcludeComposerLock(IOInterface $io)
```

Comment:
I don't think this needs to be a question. Best practice is to commit the lock file so it shouldn't be optional.